appinventor/components/src/com/google/appinventor/components/runtime/ListView.java

```diff
@@ -651,56 +651,24 @@ public void TextColorDetail(int argb) {
   }
 
   /**
-   * Returns the listview's text font Size
-   *
-   * This property is provided for backwards compatibility
-   * it takes and returns an integer, but in reality it just
-   * updates the FontSize property, which works with floats
-   *
-   * @return text size as an integer
-   */
-  @SimpleProperty(description = "The text size of the listview items.",
-      category = PropertyCategory.APPEARANCE)
```

Comment:
Mark the old versions as `@Deprecated` but leave the code on the off chance they're called by extensions.